Revit_Engine/Query/IsZero.cs

```diff
@@ -32,7 +32,8 @@ public static partial class Query
         /***************************************************/
         /****              Public methods               ****/
         /***************************************************/
-
+        
+        [DeprecatedAttribute("3.1", "This method is questionably useful and potentially harmful as it introduces a hardcoded tolerance.")]
```

Comment:
As much as I find the description funny, could we perhaps go for something slightly less abrasive to the user? :smile:

Reply:
I've heard honesty is a virtue 🐒 I tried to make it more politically correct yet informative - do you like it more after the change?